build.gradle

```
@@ -226,12 +227,15 @@ ext.libraries = [
 ]
 
 dependencies {
+  compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.7'
```

```suggestion
  implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.7'
```
And same for other new one

---

done